CRITICAL - 2022-07-16 17:43:50 --> session_start(): Failed to decode session object. Session has been destroyed
in SYSTEMPATH\Session\Session.php on line 896.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'session_start(): Failed to decode session object. Session has been destroyed', 'SYSTEMPATH\\Session\\Session.php', 896)
 2 SYSTEMPATH\Session\Session.php(896): session_start()
 3 SYSTEMPATH\Session\Session.php(240): CodeIgniter\Session\Session->startSession()
 4 SYSTEMPATH\Config\Services.php(613): CodeIgniter\Session\Session->start()
 5 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\Services::session(Object(Config\App), false)
 6 SYSTEMPATH\Config\BaseService.php(192): CodeIgniter\Config\BaseService::__callStatic('session', [...])
 7 SYSTEMPATH\Config\Services.php(585): CodeIgniter\Config\BaseService::getSharedInstance('session', null)
 8 SYSTEMPATH\Config\BaseService.php(251): CodeIgniter\Config\Services::session()
 9 SYSTEMPATH\Common.php(932): CodeIgniter\Config\BaseService::__callStatic('session', [])
10 VENDORPATH\myth\auth\src\Authentication\AuthenticationBase.php(128): session('logged_in')
11 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
12 VENDORPATH\myth\auth\src\Filters\LoginFilter.php(29): Myth\Auth\Authentication\LocalAuthenticator->check()
13 SYSTEMPATH\Filters\Filters.php(173): Myth\Auth\Filters\LoginFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), null)
14 SYSTEMPATH\CodeIgniter.php(440): CodeIgniter\Filters\Filters->run('admin', 'before')
15 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
16 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
17 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2022-07-16 18:21:05 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 718.
 1 SYSTEMPATH\BaseModel.php(718): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(666): CodeIgniter\BaseModel->insert([], false)
 3 SYSTEMPATH\BaseModel.php(658): CodeIgniter\Model->insert(Object(Myth\Auth\Entities\User), false)
 4 VENDORPATH\myth\auth\src\Controllers\AuthController.php(179): CodeIgniter\BaseModel->save(Object(Myth\Auth\Entities\User))
 5 SYSTEMPATH\CodeIgniter.php(896): Myth\Auth\Controllers\AuthController->attemptRegister()
 6 SYSTEMPATH\CodeIgniter.php(466): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2022-07-16 18:24:09 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 718.
 1 SYSTEMPATH\BaseModel.php(718): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(666): CodeIgniter\BaseModel->insert([], false)
 3 SYSTEMPATH\BaseModel.php(658): CodeIgniter\Model->insert(Object(Myth\Auth\Entities\User), false)
 4 VENDORPATH\myth\auth\src\Controllers\AuthController.php(179): CodeIgniter\BaseModel->save(Object(Myth\Auth\Entities\User))
 5 SYSTEMPATH\CodeIgniter.php(896): Myth\Auth\Controllers\AuthController->attemptRegister()
 6 SYSTEMPATH\CodeIgniter.php(466): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2022-07-16 19:09:07 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 718.
 1 SYSTEMPATH\BaseModel.php(718): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(666): CodeIgniter\BaseModel->insert([], false)
 3 SYSTEMPATH\BaseModel.php(658): CodeIgniter\Model->insert(Object(Myth\Auth\Entities\User), false)
 4 VENDORPATH\myth\auth\src\Controllers\AuthController.php(179): CodeIgniter\BaseModel->save(Object(Myth\Auth\Entities\User))
 5 SYSTEMPATH\CodeIgniter.php(896): Myth\Auth\Controllers\AuthController->attemptRegister()
 6 SYSTEMPATH\CodeIgniter.php(466): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2022-07-16 19:20:07 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 718.
 1 SYSTEMPATH\BaseModel.php(718): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(666): CodeIgniter\BaseModel->insert([], false)
 3 SYSTEMPATH\BaseModel.php(658): CodeIgniter\Model->insert(Object(Myth\Auth\Entities\User), false)
 4 VENDORPATH\myth\auth\src\Controllers\AuthController.php(179): CodeIgniter\BaseModel->save(Object(Myth\Auth\Entities\User))
 5 SYSTEMPATH\CodeIgniter.php(896): Myth\Auth\Controllers\AuthController->attemptRegister()
 6 SYSTEMPATH\CodeIgniter.php(466): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2022-07-16 19:23:11 --> There is no data to insert.
in SYSTEMPATH\BaseModel.php on line 718.
 1 SYSTEMPATH\BaseModel.php(718): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('insert')
 2 SYSTEMPATH\Model.php(666): CodeIgniter\BaseModel->insert([], false)
 3 SYSTEMPATH\BaseModel.php(658): CodeIgniter\Model->insert(Object(Myth\Auth\Entities\User), false)
 4 VENDORPATH\myth\auth\src\Controllers\AuthController.php(179): CodeIgniter\BaseModel->save(Object(Myth\Auth\Entities\User))
 5 SYSTEMPATH\CodeIgniter.php(896): Myth\Auth\Controllers\AuthController->attemptRegister()
 6 SYSTEMPATH\CodeIgniter.php(466): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
 7 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2022-07-16 19:24:39 --> Unknown column 'username' in 'where clause'
CRITICAL - 2022-07-16 19:24:39 --> Unknown column 'username' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *
FROM `siswa`
WHERE `username` = \'admin\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(695): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `siswa`
WHERE `username` = \'admin\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(609): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `siswa`
WHERE `username` = \'admin\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `siswa`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(249): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(622): CodeIgniter\Model->doFirst()
 7 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(140): CodeIgniter\BaseModel->first()
 8 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(19): Myth\Auth\Authentication\LocalAuthenticator->validate([...], true)
 9 VENDORPATH\myth\auth\src\Controllers\AuthController.php(87): Myth\Auth\Authentication\LocalAuthenticator->attempt([...], false)
10 SYSTEMPATH\CodeIgniter.php(896): Myth\Auth\Controllers\AuthController->attemptLogin()
11 SYSTEMPATH\CodeIgniter.php(466): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2022-07-16 19:27:39 --> Unknown column 'username' in 'where clause'
CRITICAL - 2022-07-16 19:27:39 --> Unknown column 'username' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *
FROM `siswa`
WHERE `username` = \'admin\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(695): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `siswa`
WHERE `username` = \'admin\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(609): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `siswa`
WHERE `username` = \'admin\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `siswa`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(249): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(622): CodeIgniter\Model->doFirst()
 7 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(140): CodeIgniter\BaseModel->first()
 8 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(19): Myth\Auth\Authentication\LocalAuthenticator->validate([...], true)
 9 VENDORPATH\myth\auth\src\Controllers\AuthController.php(87): Myth\Auth\Authentication\LocalAuthenticator->attempt([...], false)
10 SYSTEMPATH\CodeIgniter.php(896): Myth\Auth\Controllers\AuthController->attemptLogin()
11 SYSTEMPATH\CodeIgniter.php(466): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(349): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(55): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
